### PR TITLE
Update copyright header in *.ps1, *.psm1 files

### PR DESCRIPTION
**Pull request by xtqqczze**
Remove All Rights Reserved term

### CODE REVIEW

File: src/controls/animatedStatusBar.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
-/*---------------------------------------------------------
- * Copyright (C) Microsoft Corporation. All rights reserved.
- *--------------------------------------------------------*/
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation.
+ *  Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
```
Comment (TravisEz13) [inline src/controls/animatedStatusBar.ts:4]:
```suggestion
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.
```

Comment (TravisEz13) [reply src/controls/animatedStatusBar.ts:4]: Please fix in other TS files

Comment (xtqqczze) [reply src/controls/animatedStatusBar.ts:4]: I think in *.ts files block comments headers are more conventional. Also, I wanted to be consistant with microsoft/vscode, see <https://github.com/microsoft/vscode/issues/96747>.

Comment (TravisEz13) [reply src/controls/animatedStatusBar.ts:4]: This is a required style.

**Comment by xtqqczze** [comment]
@TravisEz13 I have removed changes to to *.ts files etc. These changes will be made in a seperate PR.

I am now planning to fix https://github.com/PowerShell/vscode-powershell/issues/2882 and then use [eslint-plugin-notice](https://www.npmjs.com/package/eslint-plugin-notice) to fix and enforce the correct copyright header in *.ts files.

**Pull request merged**